drivers/clock_control/Kconfig.nrf

```
@@ -56,16 +56,15 @@ endchoice
 
 config CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
 	bool "Enable LF clock calibration"
-	depends on !SOC_SERIES_NRF91X
-	depends on !CLOCK_CONTROL_NRF_FORCE_ALT
+	depends on !SOC_SERIES_NRF91X && CLOCK_CONTROL_NRF_K32SRC_RC
 	default n if BOARD_NRF5340PDK_NRF5340_CPUAPP || BOARD_NRF5340PDK_NRF5340_CPUAPPNS
 	default y if CLOCK_CONTROL_NRF_K32SRC_RC
```

```suggestion
	default y
```
This condition here became redundant.

---

Fixed